src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpRetry.cs

```diff
@@ -140,7 +142,7 @@ private static int CalculateNextRetryDelay(int nextRetryDelayMilliseconds)
             return null;
         }
 
-        var statusDetails = trailers.Get(GrpcStatusDetailsHeader);
+        var statusDetails = trailers?.Get(GrpcStatusDetailsHeader);
```

Comment:
`trailers` is not null here.

```suggestion
        var statusDetails = trailers!.Get(GrpcStatusDetailsHeader);
```